### ISSUE TITLE
process! support mutability-specifier for bindings

### ISSUE THREAD

**Issue by steffengy** [dragostis/pest#50]
What do you think about extending the ``process!``-macro to support a syntax like:
```
(&a: token, mut b: _parserXYZ()) => { ... }
```

This would essentially allow passing e.g. a vec/linked list around and modifying it in-place.

The current workaround would be forcing a move:
```
(&a: token, b: _parserXYZ()) => { let mut b_mut = b; ... }
```

**Comment by dragostis** [comment]
I'll add this right away.

**Comment by dragostis** [comment]
@steffengy I've published `0.2.3` which handles mutability the way you mentioned. It's funny how I didn't even notice this issue.
